.travis.yml

```diff
@@ -42,12 +42,24 @@ before_install:
     - export XMLLINT_INDENT="	"
     - export PHPCS_DIR=/tmp/phpcs
     - export PHPCS_BIN=$PHPCS_DIR/bin/phpcs
+    - export WPCS_DIR=/tmp/wpcs
+    - export PHPCOMPAT_DIR=/tmp/phpcompatibility
     - mkdir -p $PHPCS_DIR && git clone --depth 1 https://github.com/squizlabs/PHP_CodeSniffer.git -b $PHPCS_BRANCH $PHPCS_DIR
     - $PHPCS_BIN --config-set installed_paths $(pwd)
```

Comment:
Is this line still needed? It seems like the configuration is set twice when `$SNIFF` is configured.
When not sniffing, this does not need to be cloned or configured right?

Reply:
When not sniffing, the unit tests are run using the PHPCS unit test framework. For that to *only* test the YoastCS native sniffs, the YoastCS standard needs to be registered, so yes, this is still needed, though it could possibly be combined with the version of it below with an if/then/else syntax. You know how to do those in Travis scripts ?